Aspekt.Bootstrap/Bootstrap.cs

```diff
@@ -123,31 +114,15 @@ private static void LoadArg(InstructionHelper ic, CustomAttributeArgument arg)
                     return;
                 case MetadataType.Class:
                     // If it's a class type AND System.Type then I need to load the type. Otherwise not sure.
-                    if (arg.Type.FullName == typeof(System.Type).FullName)
+                    if (arg.Type.FullName == typeof(Type).FullName)
                     {
                         ic.Next(OpCodes.Ldtoken, (TypeReference)arg.Value);
                         ic.Call<Type>("GetTypeFromHandle", typeof(RuntimeTypeHandle));
                         return;
                     }
                     else
                         throw new Exception("unknown type");
-
-                case MetadataType.Single:
-                case MetadataType.Pointer:
-                case MetadataType.ByReference:
-                case MetadataType.Var:
-                case MetadataType.Array:
-                case MetadataType.GenericInstance:
-                case MetadataType.TypedByReference:
-                case MetadataType.IntPtr:
-                case MetadataType.UIntPtr:
-                case MetadataType.FunctionPointer:
-                case MetadataType.Object:
-                case MetadataType.MVar:
-                case MetadataType.RequiredModifier:
-                case MetadataType.OptionalModifier:
-                case MetadataType.Sentinel:
-                case MetadataType.Pinned:
```

Comment:
I'd prefer to leave these in for brevity. I'm not sure what I'm doing is right just yet. I need to read a bit more about what to do for this (if it's even possible). I'm not sure how to cherrypick and just exclude this part of the PR.